tests/pems/permutations/generate-certs.sh

```diff
@@ -168,6 +169,9 @@ then
     cert-gen   ec          ecdsa       384        SHA256      ec_ecdsa_p384_sha256
     cert-gen   ec          ecdsa       384        SHA384      ec_ecdsa_p384_sha384
     cert-gen   ec          ecdsa       521        SHA384      ec_ecdsa_p521_sha384
+    cert-gen   ec          ecdsa       521        SHA512      ec_ecdsa_p521_sha512
+    cert-gen   rsa        pkcsv1.5     2048       SHA1        rsae_pkcs_2048_sha1
```

Comment:
Kind of a nit, we usually specify _which_ pkcs padding is used. Otherwise pkcs doesn't really have much meaning on its own.

Reply:
Mostly the purpose of this is to iterate over the different NID's so pkvs vs pss is enough for my test purposes, but it sounds like a nice future item to add support for different pkcs padding choices.